docs/content/tutorials/new-app/index.md

```diff
@@ -112,7 +112,7 @@ Radius Applications are where all your app's resources and relationships come to
 
    {{% rad file="snippets/1-app.bicep" embed=true %}}
 
-1. Deploy `app.bicep` with [`rad deploy`]({{< ref rad_deploy >}}):
+2. Deploy `app.bicep` with [`rad deploy`]({{< ref rad_deploy >}}):
```

Comment:
Our markdown numbering purposefully uses `1.` for every item, as our markdown renderer automatically increments the number. Could you please keep them as `1.`?

Reply:
Changed them back to 1 for this  PR. There was some inconsistency across the docs in the numbering, but I'll change them to all to 1. in a separate PR. Thanks!